lib/elixir/src/elixir_parser.yrl

```diff
@@ -637,6 +634,10 @@ handle_literal(Literal, Token, ExtraMeta) ->
     false -> Literal
   end.
 
+integer_value(Token) ->
```

Comment:
Can we please pattern match all the way?

    integer_value({int, {_, _, Value}, _}) ->
      Value.